src/components/Forms/Artist/hooks/useArtistForm.ts

```diff
@@ -34,12 +34,12 @@ export default function useArtistForm(
     const isEditing = !!currentValues
     const defaultValues: ArtistFormValues = currentValues
         ? {
+              ...currentValues,
```

Comment:
Why are we spreading here? Looks like we should just follow the previous pattern and spell out each value with a fallback.